utils/procs/patches/001-Look-for-a-config-file-in-etc.patch

```diff
@@ -0,0 +1,26 @@
+--- a/src/main.rs
++++ b/src/main.rs
+@@ -208,12 +208,22 @@ fn get_config(opt: &Opt) -> Result<Config, Error> {
```

Comment:
```suggestion
@@ -208,12 +208,22 @@ fn get_config(opt: &Opt) -> Result<Confi
```

quilt refresh.

btw any progress with upstreaming this patch?

Reply:
sorry for the delay it has not been a great month -.-

https://github.com/dalance/procs/pull/533
^ I've created the PR, I'm waiting for the review

Reply:
It was merged and a new version was released  https://github.com/dalance/procs/releases/tag/v0.14.1